src/core/tokens/base_tokens.scss

```diff
@@ -250,10 +250,10 @@
   * @tokens Input Heights
   * @presenter Spacing
   */
-  --op-input-height-small: 2.4rem; // 24px
-  --op-input-height-medium: 3.2rem; // 32px
-  --op-input-height-large: 4.4rem; // 44px
-  --op-input-height-x-large: 5.6rem; // 56px
+  --op-input-height-small: 2.8rem; // 28px
+  --op-input-height-medium: 3.6rem; // 36px
+  --op-input-height-large: 4rem; // 40px
+  --op-input-height-x-large: 8.4rem; // 84px
```

Comment:
@Jeremy-Walton Is this one being used as a default for `textarea`?

Reply:
Not currently but that can be addressed in https://trello.com/c/Z3ThAj5l It's not currently used at all